opentelemetry-api/src/opentelemetry/util/types.py

```diff
@@ -15,5 +15,7 @@
 
 import typing
 
-AttributeValue = typing.Union[str, bool, float]
+AttributeValue = typing.Union[
+    str, bool, int, float, typing.Sequence[typing.Union[str, bool, int, float]]
```

Comment:
This is not completely correct, because the spec only allows homogeneous lists, i.e. the Union should allow:
```
str, bool, int, float, typing.Sequence[str], typing.Sequence[bool], typing.Sequence[int], typing.Sequence[float]]
```